### DISCUSSION TITLE
Timer not executing

### DISCUSSION THREAD

**Discussion by eightbit88** [arendst/Tasmota#20338] [General]
Hi all, I have a timer I try to execute once ever day with the rule, but running into some trouble.
Rule1 ON Rules#Timer=2, Do Webquery http://my_IP/externaltest POST {"from tasmota sensor"} Program finished ENDON
I can successfully execute the rule whenever I don't use the #Timer=x, I set the timer from the web interface and when I check the timer in the console, it is correctly set. All fields within the HTML timer interface are set to on e.g. Enable, enable all days...., when I issue the time command on the console it shows the correct local time. 
When I use 
Rule1 ON Time#Minute=15, Do Webquery http://my_IP/externaltest POST {"from tasmota sensor"} Program finished endon it also executes correctly.
The local time was set with Sydney Australia command in the console with Backlog0 Timezone 99; TimeStd 1,1,4,1,3,600; TimeDst 1,1,10,1,2,660
I also tried Timezone +10:00. I noticed that when it is trying to execute the Rule it seams to hang for a short time, but I'm not a 100% about that. At the moment I try to use a workaround with.
Rule1 On Time#Minute|1440, Do Webquery http://my_IP/externaltest POST {"from tasmota sensor"} Program finished endon

Any help would be greatly appreciated

**Comment by barbudor** [comment]
> Rule1 ON Rules#Timer=2, Do Webquery http://my_IP/externaltest POST {"from tasmota sensor"} Program finished ENDON

If this is an exact copy of your rule, you have a syntax error at the comma after the trigger
Remove it and you should be good

**Reply by sfromis** [reply]
As there is no minute 1440, I'd not expect `Time#Minute|1440` to work. And the Configure Timers page with a `Clock#Timer==1` (etc) rule is indeed a good way of a daily schedule. Not clear what troubles you have, but console output with `Weblog 3` is generally useful for debugging. And making sure that your rules are enabled and have all syntax details **exactly** right. You could use the output of the command `Rule0` to check, and maybe post that, to reduce the risk of things not being copied exactly.

For correct local time settings, I can recommend using the tool https://tasmotatimezone.com/ to generate a command to set all the relevant configs at the same time, including daylight savings time
